### DISCUSSION TITLE
Mouse zoom

### DISCUSSION THREAD

**Discussion by cavrelsilvera** [bpatrik/pigallery2#472] [Ideas]
I have noticed that using the mouse to zoom, the focus is always in the center of the image.  Is there a way to have that behavior changed.  Can the zoom action be focused on where the pointer of the mouse is located.

**Comment by bpatrik** [comment]
Thank might be a bug, I will need to look into it.

**Comment by cavrelsilvera** [comment]
Thank you.
Cavrel Silvera
